shared/unattended/Fedora-30.ks

```diff
@@ -0,0 +1,35 @@
+install
+KVM_TEST_MEDIUM
+GRAPHICAL_OR_TEXT
+reboot
+lang en_US
+keyboard us
+network --bootproto dhcp --hostname atest-guest
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+firstboot --disable
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+zerombr
+poweroff
+KVM_TEST_LOGGING
+
+clearpart --all --initlabel
+part / --fstype=ext4 --grow --asprimary --size=1
+
+%packages --ignoremissing
+@minimal-environment
```

Comment:
There used to be `c-development`, `sg3-utils` and others that are necessary for many tests (this is not JeOS)

Reply:
... but as mentioned earlier, IMO it's better to re-use the existing kickstart which works for everything and is proven to work with many tests.

Reply:
Yep, also caused by me adding a new kickstart file (due to none existing in the tree at the moment).